flash/text/seq2seq/summarization/utils.py

```diff
@@ -13,14 +13,12 @@
 # limitations under the License.
 import re
 
-from filelock import FileLock
 from pytorch_lightning.utilities import _module_available
 
 nltk = None
 if _module_available('nltk'):
     import nltk
-    with FileLock(".lock") as lock:
-        nltk.download("punkt", quiet=True)
+    nltk.download("punkt", quiet=True)
```

Comment:
then we can remove it also from requirements, right?

Reply:
yes! let me do this